certbot-compatibility-test/certbot_compatibility_test/test_driver.py

```diff
@@ -308,7 +306,7 @@ def get_args():
         "-e", "--enhance", action="store_true", help="tests the enhancements "
         "the plugin supports (implicitly includes installer tests)")
 
-    for plugin in PLUGINS.itervalues():
+    for plugin in PLUGINS.values():
```

Comment:
I do not think it matter. Difference between `values` and `itervalues` on Python 2.7 is about having an iterator on the map, or on a copy of it. However in the case of a copy, the references are taken as is, pointed objects are not cloned, only the list itself is cloned. It can matter on very large maps, but not on the typical size for a map of plugins.

Reply:
Ah you're right. This code is fine.